include/unifex/let_value_with_stop_source.hpp

```diff
@@ -165,7 +167,7 @@ struct _stop_source_operation<SuccessorFactory, Receiver>::type {
           nothrow_connectable) {
     return unifex::connect(
         static_cast<SuccessorFactory&&>(func)(stopSource_),
```

Comment:
This should probably be a `move`, too, which might mean it should be taken by rvalue reference.
```suggestion
        std::move(func)(stopSource_),
```

Reply:
`SuccessorFactory&&` also seems like a "saved for later forwarding reference," although it's used differently than in the other algos. Do we need `func_` to remain alive for the duration of the operation? In the operation constructor, we currently have (annotated with `// comment`)

```
template <typename SuccessorFactory2, typename Receiver2>
  type(SuccessorFactory2&& func, Receiver2&& r) noexcept(
      std::is_nothrow_constructible_v<SuccessorFactory, SuccessorFactory2>&&
          std::is_nothrow_constructible_v<Receiver, Receiver2>&& noexcept(
              connect_inner_op(func, (Receiver2 &&) r))) // borrow func, whether it's an lvalue ref or rvalue erf
    : func_{(SuccessorFactory2 &&) func} // forward func into func_
    , receiverToken_(get_stop_token(r))
    , innerOp_(connect_inner_op(func_, (Receiver2 &&) r)) {} // borrow func_, but don't move so that func_ remains alive
```

`SuccessorFactory&` in `connect_inner_op` seems correct so that it borrows but doesn't possibly move from.